test/python/tools/monitor/test_backend_monitor.py

```diff
@@ -38,6 +42,7 @@ def test_backend_overview(self, qe_token, qe_url):
         self.assertIn('Avg. T1:', stdout)
         self.assertIn('Num. Qubits:', stdout)
 
+    @unittest.skipIf(not HAS_NET_CONNECTION, "requries internet connection.")
```

Comment:
```suggestion
    @unittest.skipIf(not HAS_NET_CONNECTION, "requires internet connection.")
```

Reply:
I fixed this typo in https://github.com/Qiskit/qiskit-terra/pull/2448